content/docs/catalogue/items.md

```diff
@@ -98,6 +98,32 @@ There are two tabs in the bottom part of the screen:
 This tab lists the Master Lists associated with this item.
 ![Item detail headers](/docs/catalogue/images/cat_itemmasterlists.png)
 
+### Ledger
+
+The item ledger tab shows the history of the item.
+
+![](/docs/catalogue/images/item_ledger.png)
+
+The ledger is divided into 15 columns:
+
+| Column               | Description                                                      |
+| :------------------- | :--------------------------------------------------------------- |
+| **Type**             | The type of the transaction                                      |
+| **Date**             | The date of the transaction was confirmed                        |
+| **Time**             | The time of the transaction was confirmed                        |
+| **Invoice Number**   | The invoice number of the transaction                            |
```

Comment:
maybe?

```suggestion
| **Invoice Number**   | The invoice (shipment) number of the transaction                            |
```
 how confusing is the use of invoice here, when we have inbound shipment and outbound shipment elsewhere?